<launch>
	<arg name="box_value_" default="10"/> <!-- box 개수 -->
	<arg name="rgb_th_" default="170"/>
	

	<arg name="cols_offset_" default="0.09"/> <!-- each side value -->
	<arg name="row_offset_" default="0.72"/>
	<arg name="thresh_hold_value_" default="1200"/> <!-- detecting pixel -->

<!-- white balance area -->
	<arg name="white_offset_x_" default="1"/> 
	<arg name="white_offset_y_" default="1"/>

<!-- white balance average value -->
	<arg name="white_flag1_" default="100"/> <!-- smaller -->
	<arg name="white_flag2_" default="160"/> <!-- bigger -->

<!-- white balance Thresh hold -->
	<arg name="white_fix1_" default="70"/> <!-- flag1 -->
	<arg name="white_fix2_" default="161"/> <!-- flag2 -->
	<arg name="white_fix3_" default="110"/> <!-- else -->

	
	<arg name="u_xl_" default="512"/>
	<arg name="u_yl_" default="624"/>
	<arg name="u_xr_" default="768"/>
	<arg name="u_yr_" default="624"/>
	<arg name="d_xl_" default="128"/>
	<arg name="d_yl_" default="864"/>
	<arg name="d_xr_" default="1152"/>
	<arg name="d_yr_" default="864"/>




	<node name="crosswalk" pkg="crosswalk" type="crosswalk_node" output="screen">
		<param name="box_value" value="$(arg box_value_)"/>
		<param name="cols_offset" value="$(arg cols_offset_)"/>
		<param name="row_offset" value="$(arg row_offset_)"/>
		<param name="thresh_hold_value" value="$(arg thresh_hold_value_)"/>
		<param name="rgb_th" value="$(arg rgb_th_)"/>
		<param name="white_offset_x" value="$(arg white_offset_x_)"/>
		<param name="white_offset_y" value="$(arg white_offset_y_)"/>
		<param name="white_flag1" value="$(arg white_flag1_)"/>
		<param name="white_flag2" value="$(arg white_flag2_)"/>
		<param name="white_fix1" value="$(arg white_fix1_)"/>
		<param name="white_fix2" value="$(arg white_fix2_)"/>
		<param name="white_fix3" value="$(arg white_fix3_)"/>

		<param name="u_xl" value="$(arg u_xl_)"/>
		<param name="u_yl" value="$(arg u_yl_)"/>
		<param name="u_xr" value="$(arg u_xr_)"/>
		<param name="u_yr" value="$(arg u_yr_)"/>
		<param name="d_xl" value="$(arg d_xl_)"/>
		<param name="d_yl" value="$(arg d_yl_)"/>
		<param name="d_xr" value="$(arg d_xr_)"/>
		<param name="d_yr" value="$(arg d_yr_)"/>

	</node>
	
</launch>
